src/main/java/com/iexec/worker/feign/LoginService.java

```diff
@@ -61,19 +61,20 @@ public String login() {
         String workerAddress = credentialsService.getCredentials().getAddress();
         ECKeyPair ecKeyPair = credentialsService.getCredentials().getEcKeyPair();
 
-        String challenge = "";
+        String challenge;
```

Comment:
```suggestion
        final String challenge;
```
?

Reply:
impossible, this variable is updated in the try block

Reply:
The variable is not updated in the try block, it is assigned. As a `final` keyword means the value won't ever be updated after its first initialization, you should be able to set this variable `final`.